pkg/tracker/runner.go

```diff
@@ -31,15 +31,17 @@ func NewRunner(client rpc.ETHClient, db db.DB) (*Runner, error) {
 }
 
 // Start will kick off the runner until the given exit channel selects.
-func (r *Runner) Start(ctx context.Context, exitCh chan int) error {
+func (r *Runner) Start(ctx context.Context, logger log.Logger, exitCh chan int) error {
 	cfg := config.GetConfig()
 	trackerNames := cfg.Trackers
 	var trackers []Tracker
 	for name, activated := range trackerNames {
 		if activated {
 			t, err := createTracker(name)
 			if err != nil {
-				runnerLog.Error("Problem creating tracker: %s\n", err.Error())
+				level.Warn(logger).Log("msg", "Problem creating tracker", "name", name, "err", err)
```

Comment:
I don't think it makes sense to continue if one of the trackers fails to create.
Not everyone looks at the logs to I think this should propagate the error and the caller should decide what to to.